src/oc/web/components/user_profile.cljs

```diff
@@ -133,17 +141,13 @@
                           (rum/local false ::current-password-error)
                           {:will-mount (fn [s]
                             (dis/dispatch! [:user-profile-reset])
+                            (setup-timezone s)
```

Comment:
This is also a sign of code that should exist in the user store.  This case is a bit different because it currently uses `:input` but should probably be changed to include a timezone in the user store. As always it is possible we need the user's timezone in other components.